chain/test/Test/Pos/Chain/Txp/Json.hs

```diff
@@ -26,17 +28,51 @@ import           Test.Pos.Util.Tripping (discoverRoundTrip, roundTripsAesonShow)
 golden_TxpConfiguration0 :: Property
 golden_TxpConfiguration0 =
     goldenTestJSON exampleTxpConfiguration0
-        "test/golden/json/TxpConfiguration0"
+        "test/golden/json/TxpConfiguration0_YesNetworkMagic"
 
 golden_TxpConfiguration1 :: Property
 golden_TxpConfiguration1 =
     goldenTestJSON exampleTxpConfiguration1
-        "test/golden/json/TxpConfiguration1"
+        "test/golden/json/TxpConfiguration1_YesNetworkMagic"
 
 golden_TxpConfiguration2 :: Property
 golden_TxpConfiguration2 =
     goldenTestJSON exampleTxpConfiguration2
-        "test/golden/json/TxpConfiguration2"
+        "test/golden/json/TxpConfiguration2_YesNetworkMagic"
+
+golden_TxpConfiguration3 :: Property
+golden_TxpConfiguration3 =
+    goldenTestJSON exampleTxpConfiguration3
+        "test/golden/json/TxpConfiguration3_YesNetworkMagic"
+
+golden_TxpConfiguration4 :: Property
+golden_TxpConfiguration4 =
+    goldenTestJSON exampleTxpConfiguration4
+        "test/golden/json/TxpConfiguration4_YesNetworkMagic"
+
+golden_TxpConfiguration5 :: Property
+golden_TxpConfiguration5 =
+    goldenTestJSON exampleTxpConfiguration5
+        "test/golden/json/TxpConfiguration5_YesNetworkMagic"
```

Comment:
Of the above 6, the first 3 are the original golden tests, where the record was extended by `NMMustBeJust` (see [the modifications to the examples](https://github.com/input-output-hk/cardano-sl/pull/3422/commits/5e09477237f5f82c20471b5a670f5d6dafffcb14#diff-96d46b6ad4f97e3096f386723678d0fdR86-R96)).

The second group of 3 are new examples, which include the `NMMustBeNothing` constructor, so we have good coverage.